internal/providers/pluginfw/tfschema/attribute_builder.go

```diff
@@ -11,6 +11,8 @@ import (
 type AttributeBuilder interface {
 	BuildDataSourceAttribute() dataschema.Attribute
 	BuildResourceAttribute() schema.Attribute
+	BuildDataSourceBlock() dataschema.Block
+	BuildResourceBlock() schema.Block
```

Comment:
Not sure if this is a good idea or should we have a new interface for `BlockBuilder`